content/meetups/4.md

```diff
@@ -14,16 +14,27 @@ talks:
 
 ## ⚛️ React updates ⚛️
 
-### ⚛️ [React 16.11.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#16110-october-22-2019)
+### ️⚛️ [React Conf](https://www.youtube.com/playlist?list=PLPxbbTqCLbGHPxZpw4xj_Wwg8-fdNxJRh)
```

Comment:
Got some icon or something here? Square box

Reply:
it's the chemical reaction emoji. how does it look like from your side?

Reply:
![6710EACB-E48C-48BA-B4E4-624551687E11](https://user-images.githubusercontent.com/21194045/68381668-320dc080-018d-11ea-928d-b655356235b1.png)

Reply:
ohhh thanks for catching! fixed 🤞